LAD-AMA-Common/telegraf_utils/telegraf_config_handler.py

```diff
@@ -410,14 +413,15 @@ def parse_config(data, me_url, mdsd_url, is_lad, az_resource_id, subscription_id
     agentconf += "  \"microsoft.regionName\"= \"" + region + "\"\n"
     agentconf += "  \"microsoft.resourceId\"= \"" + az_resource_id + "\"\n"
     if virtual_machine_name != "":
-        agentconf += "  \"VMInstanceId\"= \"" + virtual_machine_name + "\"\n"
-    agentconf += "\n# Configuration for sending metrics to MetricsExtension\n"
-    agentconf += "[[outputs.influxdb]]\n"
-    agentconf += "  namedrop = [" + storage_namepass_str[:-2] + "]\n"
-    if is_lad:
-        agentconf += "  fielddrop = [" + excess_diskio_field_drop_list_str[:-2] + "]\n"
-    agentconf += "  urls = [\"" + str(me_url) + "\"]\n\n"
-    agentconf += "  udp_payload = \"2048B\"\n\n"
+        agentconf += "  \"VMInstanceId\"= \"" + virtual_machine_name + "\"\n"    
+    if has_me_output:
```

Comment:
Do we need to special-case for LAD 4 to keep metrics flowing without DCR there or we'd get it for free after LAD 4 extension config is converted into AMA/Telegraf config?

Reply:
I added a check to not change LAD behavior